packages/twenty-front/src/modules/settings/data-model/fields/preview/utils/getFieldPreviewValue.ts

```diff
@@ -1,35 +1,27 @@
 import { FieldMetadataItem } from '@/object-metadata/types/FieldMetadataItem';
-import { isFieldValueEmpty } from '@/object-record/record-field/utils/isFieldValueEmpty';
-import { generateDefaultFieldValue } from '@/object-record/utils/generateDefaultFieldValue';
+import { SettingsFieldType } from '@/settings/data-model/types/SettingsFieldType';
 import { getSettingsFieldTypeConfig } from '@/settings/data-model/utils/getSettingsFieldTypeConfig';
-import { isFieldTypeSupportedInSettings } from '@/settings/data-model/utils/isFieldTypeSupportedInSettings';
-import { isDefined } from '~/utils/isDefined';
+import { FieldMetadataType } from '~/generated-metadata/graphql';
+
+const DEFAULT_NUMBER_PREVIEW_VALUE = 2000;
```

Comment:
Here, you define something that is specific to Number.
However, this preview helper is agnositc (neutral) of its children. 

Consequence is that if you look at an adress preview, it is now broken. Maybe you can find a way to make it ok for all kind of fieldMetadataItem types ?

Reply:
alright, will look into this